apps/cowswap-frontend/src/common/hooks/useSafeMemo.ts

```diff
@@ -1,13 +1,22 @@
 import { EffectCallback, useEffect, useMemo } from 'react'
 
-import { CurrencyAmount, NativeCurrency, Percent, Token } from '@uniswap/sdk-core'
+import { CurrencyAmount, NativeCurrency, Percent, Price, Token } from '@uniswap/sdk-core'
 
 export function useSafeDeps(deps: unknown[]): unknown[] {
   return deps.map((dep) => {
-    if (dep instanceof NativeCurrency) return dep.symbol
-    if (dep instanceof Token) return dep.address.toLowerCase()
+    if (dep instanceof NativeCurrency) return dep.symbol || '' + dep.chainId
```

Comment:
I think you need the parenthesis

<img width="229" alt="image" src="https://github.com/user-attachments/assets/af6fd310-c546-47d7-b8a2-caa41e99eb47" />

Reply:
98f872524f91